openjdk-11-jre-headless fails to install

We were looking for `openjdk-17-jre-headless` but noticed that `openjdk-11-jre-headless` is version `17.0.5.8-r4`, however, given,

```
contents:
  keyring:
    - https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
  repositories:
    - https://packages.wolfi.dev/os
  packages:
    - wolfi-base
    - openjdk-11-jre-headless

entrypoint:
  command: /bin/sh -l

archs:
  - x86_64
```

We see the error,

```
docker run -v ./docker:/work cgr.dev/chainguard/apko build /work/base-apko.yaml my-base:edge /work/mybase.tar
2024/05/29 19:23:07 INFO Building images for 1 architectures: [amd64]
2024/05/29 19:23:10 INFO installing fontconfig-config (2.15.0-r1) arch=x86_64
2024/05/29 19:23:10 INFO installing ca-certificates-bundle (20240315-r3) arch=x86_64
2024/05/29 19:23:10 INFO installing wolfi-baselayout (20230201-r11) arch=x86_64
2024/05/29 19:23:10 INFO installing ld-linux (2.39-r6) arch=x86_64
2024/05/29 19:23:10 INFO installing glibc-locale-posix (2.39-r6) arch=x86_64
2024/05/29 19:23:11 INFO installing glibc (2.39-r6) arch=x86_64
2024/05/29 19:23:11 INFO installing libexpat1 (2.6.2-r1) arch=x86_64
2024/05/29 19:23:11 INFO installing libbz2-1 (1.0.8-r7) arch=x86_64
2024/05/29 19:23:11 INFO installing libbrotlicommon1 (1.1.0-r2) arch=x86_64
2024/05/29 19:23:11 INFO installing libbrotlidec1 (1.1.0-r2) arch=x86_64
2024/05/29 19:23:11 INFO installing zlib (1.3.1-r2) arch=x86_64
2024/05/29 19:23:11 INFO installing libpng (1.6.43-r2) arch=x86_64
2024/05/29 19:23:11 INFO installing freetype (2.13.2-r4) arch=x86_64
2024/05/29 19:23:11 INFO installing libfontconfig1 (2.15.0-r1) arch=x86_64
2024/05/29 19:23:11 INFO installing fontconfig (2.15.0-r1) arch=x86_64
2024/05/29 19:24:24 INFO installing openjdk-11-jre-headless (17.0.5.8-r4) arch=x86_64
2024/05/29 19:24:24 INFO installing wolfi-keys (1-r8) arch=x86_64
2024/05/29 19:24:24 INFO installing libcrypto3 (3.3.0-r7) arch=x86_64
2024/05/29 19:24:24 INFO installing libssl3 (3.3.0-r7) arch=x86_64
2024/05/29 19:24:24 INFO installing apk-tools (2.14.1-r2) arch=x86_64
2024/05/29 19:24:24 INFO installing libxcrypt (4.4.36-r7) arch=x86_64
2024/05/29 19:24:24 INFO installing libcrypt1 (2.39-r6) arch=x86_64
2024/05/29 19:24:24 INFO installing busybox (1.36.1-r10) arch=x86_64
2024/05/29 19:24:24 INFO installing wolfi-base (1-r6) arch=x86_64
2024/05/29 19:24:26 INFO built image layer tarball as /tmp/apko-temp-2759762063/apko-x86_64.tar.gz arch=x86_64
2024/05/29 19:24:26 INFO OCI layer digest: sha256:b1be4883b7580591535ccf22e81969163d61bfdcca6e2c7872665bf26924bd0b arch=x86_64
2024/05/29 19:24:26 INFO OCI layer diffID: sha256:e2b3cfd6650716a87209bdc85a5e488c4272a89b30cc9675b5d3cb958647bf59 arch=x86_64
2024/05/29 19:24:26 INFO built index file as /tmp/apko-temp-2759762063/index.json
Error: generating sbom for amd64: generating spdx sbom: parsing internal apk SBOM: copying element: unable to find 109 elements in source document: [SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.base/unicode.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.xml/jcup.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libfontmanager.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libjawt.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/server/libjsig.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/bin/keytool SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libjsig.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/src.zip SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.base/ADDITIONAL_LICENSE_INFO SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.desktop/mesa3d.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.desktop/xwd.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libjavajpeg.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.base/cldr.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.xml/dom.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libjli.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/conf/security/policy/unlimited/default_local.policy SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/jfr/profile.jfc SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libsyslookup.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/psfont.properties.ja SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/psfontj2d.properties SPDXRef-File-/usr/lib/jvm/openjdk-jre/conf/security/policy/limited/default_US_export.policy SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/security/cacerts SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/server/classes.jsa SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libmanagement.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/bin/jrunscript SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/jdk.internal.le/jline.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/classlist SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libj2pcsc.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libmanagement_ext.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/bin/java SPDXRef-File-/usr/lib/jvm/openjdk-jre/bin/jfr SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.xml/xerces.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/conf/security/policy/unlimited/default_US_export.policy SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libj2gss.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libjava.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/security/blocked.certs SPDXRef-File-/usr/lib/jvm/openjdk-jre/LICENSE SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.desktop/colorimaging.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/release SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.xml/xalan.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/conf/sound.properties SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/jdk.javadoc/jqueryUI.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libjsvml.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libverify.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/conf/management/jmxremote.access SPDXRef-File-/usr/lib/jvm/openjdk-jre/conf/management/jmxremote.password.template SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.xml/bcel.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/jdk.crypto.cryptoki/pkcs11cryptotoken.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/jdk.dynalink/dynalink.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/jdk.internal.opt/jopt-simple.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/ASSEMBLY_EXCEPTION SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.base/c-libutl.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libmanagement_agent.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libnet.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/conf/security/policy/limited/exempt_local.policy SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.base/ASSEMBLY_EXCEPTION SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.base/icu.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libawt_headless.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/librmi.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/jdk.crypto.cryptoki/pkcs11wrapper.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/server/libjvm.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.smartcardio/pcsclite.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libdt_socket.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libjaas.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libprefs.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/conf/security/policy/limited/default_local.policy SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.base/aes.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libnio.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/modules SPDXRef-File-/usr/lib/jvm/openjdk-jre/conf/logging.properties SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libmlib_image.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libzip.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/conf/net.properties SPDXRef-File-/usr/lib/jvm/openjdk-jre/conf/management/management.properties SPDXRef-File-/usr/lib/jvm/openjdk-jre/conf/sdp/sdp.conf.template SPDXRef-File-/usr/lib/jvm/openjdk-jre/conf/security/policy/README.txt SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.base/asm.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.desktop/harfbuzz.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libawt.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/README.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/jexec SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/jdk.javadoc/jquery.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.xml.crypto/santuario.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/jvm.cfg SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libjsound.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/liblcms.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.base/LICENSE SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libattach.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libawt_xawt.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libsctp.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/jspawnhelper SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/jrt-fs.jar SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libjimage.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/jdk.localedata/thaidict.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.base/public_suffix.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/bin/rmiregistry SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libj2pkcs11.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libjdwp.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libsaproc.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libsplashscreen.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/security/public_suffix_list.dat SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/tzdb.dat SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libextnet.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/conf/security/java.security SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/jfr/default.jfc SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libinstrument.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/security/default.policy SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/server/classes_nocoops.jsa SPDXRef-File-/usr/lib/jvm/openjdk-jre/conf/security/java.policy]
2024/05/29 19:24:26 INFO error during command execution: generating sbom for amd64: generating spdx sbom: parsing internal apk SBOM: copying element: unable to find 109 elements in source document: [SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.base/unicode.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.xml/jcup.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libfontmanager.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libjawt.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/server/libjsig.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/bin/keytool SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libjsig.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/src.zip SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.base/ADDITIONAL_LICENSE_INFO SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.desktop/mesa3d.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.desktop/xwd.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libjavajpeg.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.base/cldr.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.xml/dom.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libjli.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/conf/security/policy/unlimited/default_local.policy SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/jfr/profile.jfc SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libsyslookup.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/psfont.properties.ja SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/psfontj2d.properties SPDXRef-File-/usr/lib/jvm/openjdk-jre/conf/security/policy/limited/default_US_export.policy SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/security/cacerts SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/server/classes.jsa SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libmanagement.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/bin/jrunscript SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/jdk.internal.le/jline.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/classlist SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libj2pcsc.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libmanagement_ext.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/bin/java SPDXRef-File-/usr/lib/jvm/openjdk-jre/bin/jfr SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.xml/xerces.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/conf/security/policy/unlimited/default_US_export.policy SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libj2gss.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libjava.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/security/blocked.certs SPDXRef-File-/usr/lib/jvm/openjdk-jre/LICENSE SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.desktop/colorimaging.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/release SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.xml/xalan.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/conf/sound.properties SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/jdk.javadoc/jqueryUI.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libjsvml.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libverify.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/conf/management/jmxremote.access SPDXRef-File-/usr/lib/jvm/openjdk-jre/conf/management/jmxremote.password.template SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.xml/bcel.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/jdk.crypto.cryptoki/pkcs11cryptotoken.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/jdk.dynalink/dynalink.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/jdk.internal.opt/jopt-simple.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/ASSEMBLY_EXCEPTION SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.base/c-libutl.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libmanagement_agent.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libnet.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/conf/security/policy/limited/exempt_local.policy SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.base/ASSEMBLY_EXCEPTION SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.base/icu.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libawt_headless.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/librmi.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/jdk.crypto.cryptoki/pkcs11wrapper.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/server/libjvm.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.smartcardio/pcsclite.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libdt_socket.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libjaas.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libprefs.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/conf/security/policy/limited/default_local.policy SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.base/aes.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libnio.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/modules SPDXRef-File-/usr/lib/jvm/openjdk-jre/conf/logging.properties SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libmlib_image.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libzip.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/conf/net.properties SPDXRef-File-/usr/lib/jvm/openjdk-jre/conf/management/management.properties SPDXRef-File-/usr/lib/jvm/openjdk-jre/conf/sdp/sdp.conf.template SPDXRef-File-/usr/lib/jvm/openjdk-jre/conf/security/policy/README.txt SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.base/asm.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.desktop/harfbuzz.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libawt.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/README.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/jexec SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/jdk.javadoc/jquery.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.xml.crypto/santuario.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/jvm.cfg SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libjsound.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/liblcms.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.base/LICENSE SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libattach.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libawt_xawt.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libsctp.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/jspawnhelper SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/jrt-fs.jar SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libjimage.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/jdk.localedata/thaidict.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/legal/java.base/public_suffix.md SPDXRef-File-/usr/lib/jvm/openjdk-jre/bin/rmiregistry SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libj2pkcs11.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libjdwp.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libsaproc.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libsplashscreen.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/security/public_suffix_list.dat SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/tzdb.dat SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libextnet.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/conf/security/java.security SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/jfr/default.jfc SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/libinstrument.so SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/security/default.policy SPDXRef-File-/usr/lib/jvm/openjdk-jre/lib/server/classes_nocoops.jsa SPDXRef-File-/usr/lib/jvm/openjdk-jre/conf/security/java.policy]
make: *** [build-tar] Error 1
```

--- a/withdrawn-packages.txt
+++ b/withdrawn-packages.txt
@@ -70,3 +70,4 @@ valkey-cli-7.2.5-r0.apk
 valkey-benchmark-7.2.5-r0.apk
 istio-install-cni-1.21-1.22.0-r0.apk
 istio-install-cni-1.21-1.22.0-r1.apk
+openjdk-11-jre-headless-17.0.5.8-r4.apk

